SQLite3/module.modulemap

```diff
@@ -0,0 +1,5 @@
+module SQLite3 [system] {
+    header "/usr/include/sqlite3.h"
```

Comment:
Not against this. Was just hoping to avoid a solution like this that runs a script to adjust the path if needed: 

https://github.com/svdo/swift-netutils/tree/master/ifaddrs

Unless you can think of a better way?